src/views/signup.hbs

```diff
@@ -1,23 +1,25 @@
-<section>
-  <h1>Welcome to Well-Bean</h1>
-  <article>
-    <h3>Signup Here</h3>
-    <form action="/signup" method="POST">
+<section class="section__signup">
+  <h1 class="signup__title">Sign Up</h1>
+  <div class="signup__desc">
+  <p>We take our promise of anonymity seriously. We will never share your information with others. </p>
+  </div>
+  <article class="signup__form--container">
```

Comment:
BEM syntax should be block__element--modifier
so if you want to nest elements you should put __ between them